nuxeo-core/nuxeo-core-convert-api/src/main/java/org/nuxeo/ecm/core/convert/api/ConversionService.java

```diff
@@ -54,14 +54,17 @@ BlobHolder convert(String converterName, BlobHolder blobHolder, Map<String, Seri
     /**
      * Converts a Blob given a target destination MimeType.
      */
-    BlobHolder convertToMimeType(String destinationMimeType, BlobHolder blobHolder, Map<String, Serializable> parameters)
-            throws ConversionException;
+    BlobHolder convertToMimeType(String destinationMimeType, BlobHolder blobHolder,
+            Map<String, Serializable> parameters) throws ConversionException;
 
     /**
      * Converts a Blob to PDF. If the blob has inner blobs such as images, they will be correctly rendered in the PDF.
      *
      * @since 9.1
+     * @deprecated since 9.2, use {@link #convertToMimeType(String, BlobHolder, Map)} with the PDF mimetype as
+     *             destination instead
      */
+    @Deprecated
```

Comment:
As the fix introducing this method was backported, it was kept for compatibility.